src/common/api-interface.ts

```diff
@@ -329,7 +333,14 @@ export interface ICallNotificationData {
   shouldDisplayBadge: boolean;
   acceptButtonText: string;
   rejectButtonText: string;
+  isFederatedEnabled?: boolean;
+  isPhone?: boolean;
+  notificationType?: string;
+  isPrimaryTextOverflowing?: boolean;
+  isSecondaryTextOverflowing?: boolean;
```

Comment:
Is this needed here? initially, this is just a local component state

Reply:
Thats correct. Might be confused by the model